center_x_offset: -20
head_pan_offset: 12
